controllers/comments.controller.js

```diff
@@ -0,0 +1,11 @@
+const { insertCommentToArticle } = require("../models/comments.model.js")
+
+exports.addCommmentToArticle = (req, res, next) => {
+    const { article_id } = req.params;
+
+    insertCommentToArticle(req.body, article_id)
+    .then((comment) => {
+        res.status(201).send( { comment } );
```

Comment:
Express will never send a response body with a status 201, so we can either use `res.sendStatus(201)` , or `send()`with no arguments instead - otherwise this may lead to confusion as you might expect to be able to access the comment (which you cannot!)